### ISSUE TITLE
shorten requests need to select/focus text so that it can be copied easily

### ISSUE THREAD

**Issue by scr** [PaulKinlan/WebIntents#161]
I tried to do .select() & .focus() on a text element, but the focus is not honored unless I first click on the intent tab's content.  I'm wondering if this is a bug in the canary webintents...

**Comment by PaulKinlan** [comment]
Can you set tabindex attribute?
On Feb 18, 2012 6:43 PM, "scr" <
reply@reply.github.com>
wrote:

**Comment by scr** [comment]
yep - tried it - something in chrome (i.e. not the content) seems to be in focus - tabbing goes through each of the bookmarks before hitting the fields on the page.
